exporter/collector/googlemanagedprometheus/monitoredresource.go

```diff
@@ -69,9 +72,24 @@ func (c Config) MapToPrometheusTarget(res pcommon.Resource) *monitoredrespb.Moni
 // getStringOrEmpty returns the value of the first key found, or the empty string.
 func getStringOrEmpty(attributes pcommon.Map, keys ...string) string {
 	for _, k := range keys {
-		if val, ok := attributes.Get(k); ok {
+		if val, ok := attributes.Get(k); ok && !strings.HasPrefix(val.Str(), unknownServicePrefix) {
```

Comment:
nit can you add a comment explaining why we skip here?

Reply:
added